docs/jwst/emicorr/arguments.rst

```diff
@@ -25,3 +25,6 @@ The ``emicorr`` step has the following step-specific arguments.
     This is to tell the code to do correction for the frequencies in
     the list with a reference file created on-the-fly instead of CRDS.
 
+``--nints_to_phase`` (integer, default=None)
+    Use N cycles to calculate the phase, to use all integrations set to None.
```

Comment:
If the `nints_to_phase` param gives the number of *integrations* to use, the explanation here should not say *cycles*.